draft-ietf-tls-tls13.md

```diff
@@ -1018,8 +1027,9 @@ https://github.com/tlswg/tls13-spec/issues/443]]
 IMPORTANT NOTE: The security properties for 0-RTT data are weaker than
 those for other kinds of TLS data.  Specifically:
 
-1. This data is not forward secret, because it is encrypted solely
-with the PSK.
+1. The early data is encrypted solely under keys derived using the 
+offered PSK as the static secret. Hence, this data is not forward 
+secret, because it is encrypted solely with the PSK.
```

Comment:
This repeats the same thought twice. ("solely" twice with different wording) If we want to add the wording of the new sentence here, then a suggested replacement:

"
1. This data is not forward secret, as it is encrypted solely under keys derived using the offered PSK as the static secret.
"